### PR TITLE
Replace (3) and (7) with (3scheme) and (7scheme)

### PR DESCRIPTION
**Pull request by lassik**

### CODE REVIEW

File: man3/exit.3scheme [hunk 1]
```diff
@@ -94,7 +94,7 @@ or fail catastrophically.
 .SH "SEE ALSO"
 .
 .BR emergency-exit (3scheme),
-.BR exit (3),
+.BR exit (3scheme),
```
Comment (weinholt) [inline man3/exit.3scheme]: This one is really talking about exit(3).

File: man3/finite?.3scheme [hunk 2]
```diff
@@ -148,7 +148,7 @@ with names that coincide with those in R6RS.
 .PP
 The C99 language and POSIX offer a more detailed classification which
 includes subnormals, see
-.BR fpclassify (3).
+.BR fpclassify (3scheme).
```
Comment (weinholt) [inline man3/finite?.3scheme]: This is talking about fpclassify(3).

**Comment by weinholt** [comment]
Two collateral victims in the search-and-replace, but otherwise looks ok.

**Comment by lassik** [comment]
Good catch. Fixed.

**Pull request merged**